script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -407,26 +409,26 @@ I'm not kidding when I say that your eyes really are that bad!"
 [sha:d76c3b3073f73a30dcb5bccd8a97d22e37b1ee55]{
 -- Page 594, Offset 7160.
 -- 「いい？　消滅は万物に予め設計された決まりなの。
-"Listen. Everything in this world is predestined for extinction.
+"Listen. All things are arranged in a way where their demise is determined in advance.
```

Comment:
Unless this is some lore point, I prefer the original. I see below you specified that the _agency_ is important, and if that is truly the case, then go with Yusuke's version.

Reply:
This line is so hard, honestly.

I wanted it to be as clear as it could be for the original idea. That the universe has a rule that all things must have demise determined before it is even born

Reply:
I think the original gets that across pretty clearly

Reply:
I will change extinction to demise at least